ui/main/src/app/modules/processmonitoring/processmonitoring.component.ts

```diff
@@ -238,25 +239,26 @@ export class ProcessMonitoringComponent implements OnDestroy, OnInit, AfterViewI
         this.changeStatesWhenSelectProcess();
 
         this.size = ConfigService.getConfigValue('processMonitoring.filters.page.size', 10);
-        this.tags = ConfigService.getConfigValue('processMonitoring.filters.tags.list');
-        this.page = 1;
-        this.results = [];
+        OpfabAPIService.businessconfig.getTags('processMonitoring').then((customTags) => {
+            this.tags = customTags ?? ConfigService.getConfigValue('processMonitoring.filters.tags.list');
 
-        this.tagsMultiSelectOptions = this.tags.map((tag) => {
-            return new MultiSelectOption(tag.value, tag.label);
+            this.tagsMultiSelectOptions = this.tags.map((tag) => {
+                return new MultiSelectOption(tag.value, tag.label);
+            });
+            const tagsSelectedInStorage = UserPreferencesService.getPreference('opfab.processMonitoring.tagsSelected');
+            if (tagsSelectedInStorage?.length > 0) {
+                this.tagsSelected = tagsSelectedInStorage.split(',');
+                this.processMonitoringForm.patchValue({tags: this.tagsSelected});
```

Comment:
You need to call here `this.changeDetector.markForCheck()` otherwise if tags takes time to be loaded it is not visible on screen 

I tested introducing delay in the getTags : 

```

async function getTags(screenName ) {
   let tags;
   
   if (screenName === 'processMonitoring') {
      console.log(new Date().toISOString(), 'Custom tags for processMonitoring screen');
      await new Promise(resolve => setTimeout(resolve, 3000)); 
      console.log(new Date().toISOString(),'Custom tags loaded');  
      tags = [{
         "label": "Label for custom tasdfsf sdf g 1",
         "value": "custom1"
      },
      {
         "label": "Label for custom tag 2",
         "value": "custom2"
      }];
   }
   return tags;
};
```